src/Core/Dirt/Reports/ReportFeatures/AddOrganizationReportCommand.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Bit.Core.Dirt.Entities;
+using Bit.Core.Dirt.Reports.ReportFeatures.Interfaces;
+using Bit.Core.Dirt.Reports.ReportFeatures.Requests;
+using Bit.Core.Dirt.Repositories;
+using Bit.Core.Exceptions;
+using Bit.Core.Repositories;
+
+namespace Bit.Core.Dirt.Reports.ReportFeatures;
+
+public class AddOrganizationReportCommand : IAddOrganizationReportCommand
+{
+    private readonly IOrganizationRepository _organizationRepo;
+    private readonly IOrganizationReportRepository _organizationReportRepo;
+
+    public AddOrganizationReportCommand(
+        IOrganizationRepository organizationRepository,
+        IOrganizationReportRepository organizationReportRepository)
+    {
+        _organizationRepo = organizationRepository;
+        _organizationReportRepo = organizationReportRepository;
+    }
+
+    public async Task<OrganizationReport> AddOrganizationReportAsync(AddOrganizationReportRequest request)
+    {
+        var (req, IsValid, errorMessage) = await ValidateRequestAsync(request);
```

Comment:
❓Do we need to return the `AddOrganizationReportRequest` back from the `ValidateRequestAsync()` method? We don't do anything with `req` and only use the `request` parameter in this code.